apps/backend/src/main/java/ch/heig/pdg/backend/presentation/AuthenticationController.java

```diff
@@ -30,7 +30,7 @@ public AuthenticationController(UserRepository userRepository, JWTGeneratorServi
 
     @Override
     public ResponseEntity<JWTDTO> getToken(CredentialsDTO credentialsDTO) {
-        User user = this.userRepository.findByUsername(credentialsDTO.getUsername()).orElseThrow(NotFoundException::new);
+        User user = this.userRepository.findByUsername(credentialsDTO.getUsername()).orElseThrow(UnauthorizedException::new);
```

Comment:
As per said above `orElseThrow(() -> new UnauthorizedException("wrong user")`

Reply:
No

Reply:
Both are correct.

Must of the time, the _frontend_ does not care to know why the login failed (our case).

But depending on the expected "login-flow", the reason should be sent.
EG: _Google_ or _Microsoft_ = first the email -> the password

For this project, KIS (Keep It Simple).
(Return me if the password is already used)